content/manuals/docker-hub/image-library/catalogs.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Explore specialized Docker Hub collections like the Generative AI catalog.
+keywords: Docker Hub, Hub, catalog
+title: Docker Hub catalogs
+linkTitle: Catalogs
+weight: 60
+---
+
+Docker Hub catalogs are your go-to collections of trusted, ready-to-use
+container images and resources, tailored to meet specific development needs.
+They make it easier to find high-quality, pre-verified content so you can
+quickly build, deploy, and manage your applications with confidence. Catalogs in
+Docker Hub:
+
+- Simplify content discovery: Organized and curated content makes it easy to
+  discover tools and resources tailored to your specific domain or technology.
+- Reduce complexity: Trusted resources, vetted by Docker and its partners,
+  ensure security, reliability, and adherence to best practices.
+- Accelerate development: Quickly integrate advanced capabilities into your
+  applications without the hassle of extensive research or setup.
+
+The generative AI catalog is the first catalog in Docker Hub, offering
+specialized content for AI development.
+
+## Generative AI catalog
+
+The [generative AI catalog](https://hub.docker.com/catalogs/gen-ai) makes it
+easy to explore and add AI capabilities to your applications. With trusted,
+ready-to-use content and comprehensive documentation, you can skip the hassle of
+sorting through countless tools and configurations. Instead, focus your time and
+energy on creating innovative AI-powered applications.
+
+The generative AI catalog provides a wide range of trusted content, organized
+into key areas to support diverse AI development needs:
+
+- Demos: Ready-to-deploy examples showcasing generative AI capabilities. These
+  demos provide a hands-on way to explore AI tools and frameworks, making it
+  easier to understand how they can be integrated into real-world applications.
+- Models: Pre-trained AI models for tasks like text generation,
+  natural language processing (NLP), and conversational AI. These models
```

Comment:
This deals with the vale nag

```suggestion
  Natural Language Processing (NLP), and conversational AI. These models
```

Reply:
@dvdksn Fixing it. But, is that a vale limitation, or do we consider all terms to be proper nouns if they have an acronym?

For example in this particular case, Wikipedia considers NLP not a proper noun https://en.wikipedia.org/wiki/Natural_language_processing
And Microsoft style guide says don't capitalize unless a proper noun, https://learn.microsoft.com/en-us/style-guide/acronyms#lowercase-the-spelled-out-term

Reply:
It's how our rule for these acronyms is configured:

https://github.com/docker/docs/blob/aae93f671a3799f3e6ceeca199d6041f1023529b/_vale/Docker/Acronyms.yml#L8

We only consider an acronym spelled out if it's preceded by a Proper Noun. It doesn't actually check that that the acronym and the proper nouns are the same. (In other words, Some Random Word (FOO) would be considered valid by this vale rule.)

I don't think there's a way to make this rule smarter with the built-in vale styles. But we might be able to write a [tengo script](https://vale.sh/docs/topics/styles/#script). I can take a quick look. Otherwise, I think our alternatives are:

- Use proper nouns when spelling out an acronym anyway (I don't mind this rule)
- Don't check for unexpanded acronyms at all
- Manually ignore this rule: `<!-- vale Docker.Acronyms = NO -->`